go/ql/test/library-tests/semmle/go/concepts/LoggerCall/main.go

```diff
@@ -3,6 +3,8 @@ package main
 const fmt = "formatted %s string"
 const text = "test"
 
-func main() {
+var v []byte
 
+func main() {
+	stdlib()
```

Comment:
What's the motivation for this change?

Reply:
To remove linter warnings about lots of unused things 😆 .